webapp/components/product-info.js

```diff
@@ -97,6 +109,13 @@ function parseProduct(name) {
 const ProductInfo = (superClass) => class extends superClass {
   static get properties() {
     return {
+      // Polymer templates can only access variables in the scope of the owning
+      // class. Forward some declarations so that subclasses can use them in
+      // template parameters.
+      allProducts: {
+        type: Array,
+        value: AllProducts,
```

Comment:
Nit: mark this as `readOnly: true` to prevent accidental modification.

Reply:
Nice, didn't know that was an available property. Done!